geostore/migrations/0034_auto_20191209_0902.py

```diff
@@ -1,22 +0,0 @@
-# Generated by Django 2.2.7 on 2019-12-09 09:02
```

Comment:
Problem with the migration 0034_auto_20191209_0902.py, it should not be squash with layerrelations migrations.